src/ess/reduce/live/raw.py

```diff
@@ -154,25 +154,32 @@ class LogicalView:
         corresponds to multiple dimensions in the logical view.
     transpose:
         Dimensions to transpose. This is useful for reordering dimensions.
+        Applied after sum but before flatten operations.
     select:
         Dimensions with associated index to select from the data. This extracts a slice
         of the data for each given dimension.
+    sum:
+        Dimensions to sum over after folding and selecting, but before transposing and
+        flattening. If None, sum over all dimensions.
```

Comment:
This seems quite complicated, especially because `LogicalView` uses these operations in a fixed order which may be surprising to a user. Did you consider using a user-defined function instead? The operations performed here are all quite simple to express using Scipp. So a function wouldn't be much more verbose. To clarify, I mean something like this:
```python
logical_view = LogicalView(lambda da: da.['a', 0].sum('c').transpose(('d', 'b')))
```

Reply:
> Would people also want mean instead of sum ...?

One more reason to let users provide a function.

Reply:
Works for now as is. Unless you have a good reason for wanting to change it?

Reply:
I think we should keep it just as it is since we will have another service that allows arbitrary workflow to run right...?

Reply:
Just keep it in mind for when you add the next operation.

Reply:
Well, probably `LogicalView` should just be removed, and the `projection` argument of `RollingDetectorView` can take any callable instead.